app/api-v1/controllers/Attachment/index.js

```diff
@@ -18,9 +18,25 @@ const returnOctet = ({ filename, binary_blob }) => ({
   response: binary_blob,
 })
 
+const getSize = (blob) => {
+  return Buffer.from(JSON.stringify(blob)).length.toString()
+}
+
 module.exports = {
-  get: async function () {
-    return { status: 500, response: { message: 'Not Implemented' } }
+  getAll: async function () {
```

Comment:
aren't we using `get` now rather than `getAll`? based off of changes in https://github.com/digicatapult/inteli-api/pull/56/files

Reply:
Yep, changed